### ISSUE TITLE
when running some kbox git commands, I get: error: There was a problem with the editor 'editor'.

### ISSUE THREAD

**Issue by labboy0276** [kalabox/kalabox-issues#471]
As the message states, it seems either nano/vim (I can't remember which one you chose) is not possibly configured correctly in the container.

When I run commands like kbox git revert or kbox git commit --amend, i get this:

```
john@kalamuna:~/kalabox/apps/icrw$ kbox git revert 74c36769609f2edfe63023d382b9e5015e76c67d
info: DOCKER => Creating RUN container.
info: DOCKER => Attaching RUN container.
info: DOCKER => Starting RUN container.
Identity added: /ssh/.ssh/id_rsa (/ssh/.ssh/id_rsa)
error: cannot run editor: No such file or directory
error: There was a problem with the editor 'editor'.
Please supply the message using either -m or -F option.
info: DOCKER => RUN container shutting down.
```

I can use the -m on the kbox commit --amend -m "YOUR MOM", but not on revert.  However outside of kbox, it just goes to my editor which happens to be nano.

Thoughts?

**Comment by pirog** [comment]
I think this is probably a regression. We moved rsync and git into the debian container and probably left nano behind. So for this we will want to make sure we add nano into the debian container or just the git container.

**Comment by bcauldwell** [comment]
I'm getting this as well, but haven't been able to find out what caused the regression. Is anyone able to confirm this used to work?

**Comment by pirog** [comment]
The regression was caused when we rebased the drush container on top of the php appserver container and lost NANO

**Comment by pirog** [comment]
this should be resolved in 0.8.0. you will need to make sure you update your apps to 0.8.0 and then run `kbox update` inside your app.
